network/transport/grpc/stats.go

```diff
@@ -92,6 +93,25 @@ func (o ownPeerIDStatistic) String() string {
 	return o.peerID.String()
 }
 
+// lastCertificateValidation contains the Unix timestamp of the most recent certificate validation of all peers.
+type lastCertificateValidation struct {
```

Comment:
It's a statistic, maybe the name should reflect it

Reply:
now `lastCertificateValidationStatistic`